app/models/staged/petition_creator/replay_petition.rb

```diff
@@ -1,7 +1,6 @@
 module Staged
   module PetitionCreator
     class ReplayPetition < Staged::Base::Petition
-      include Staged::PetitionCreator::HasCreatorSignature
 
       class CreatorSignature < Staged::Base::CreatorSignature
```

Comment:
If we don't include the ``Staged::PetitionCreator::HasCreatorSignature`` anymore, we also don't need the ``Staged::PetitionCreator::ReplayPetition::CreatorSignature``.